packages/components/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cabin-design-system",
+  "name": "@cabindao/topo",
```

Comment:
`topo` was already taken, plus I think it's good practice for an org to namespace all of their internal packages

Reply:
Good call